### DISCUSSION TITLE
osu!lazer wont open

### DISCUSSION THREAD

**Discussion by MERTPRO45YT** [ppy/osu#30661] [Q&A]
Hello when i try to open osu!lazer it wont open and when i look at task manager the game uses the cpu for a bit and then stops using and closes itself. The game's window wont even show and i cant do something. I tried to restart my pc, tried to close some apps that thinking maybe some apps dont let the game open and even tried the compatibility thing and it didnt work either. This started happening when i restarted my computer and i didnt reinstall the game because im scared of my beatmaps gonna be deleted. Please help me

**Comment by nagatsuka-shuuya** [comment]
https://github.com/ppy/osu/issues/30656

**Comment by peppy** [comment]
If you're on linux, then wait for the next release and update to that. If you're not, please provide more details.
